docker-compose.yml

```diff
@@ -3,8 +3,8 @@ version: '3'
 services:
   db:
     image: postgres:12.0
-    ports:
-     - "5432:5432"
+    expose:
+      - 5432
```

Comment:
Ports exposes the postgress port outside the container / network and causes a collision when building on Ubuntu, moving to expose which only opens to the docker-container network solves this.

Reply:
@William-Blackie , with `ports: - "5432:5432"`, I am able to connect to `pgAdmin4`, preferred interface, to access the database layer. Is there any other way, or have one specific for `windows`?

Reply:
Good point, I wasn't aware of that usecase, I'll revert this.